tests/src/test/scala/whisk/core/controller/test/SequenceApiTests.scala

```diff
@@ -53,6 +49,22 @@ class SequenceApiTests extends ControllerTestCommon with WhiskActionsApi {
 
   val allowedActionDuration = 120 seconds
 
+  it should "produce proper error when sequence component is missing during sequence invocation" in {
+    implicit val tid = transid()
+    val seqName = s"${aname()}_seq"
+    val compName = s"${aname()}_comp"
```

Comment:
the fix now looks correct above - can you improve the test however: the sequence should have more than one component and the result should show that the sequence partially activated.

Reply:
Hmm, I think we need to create a integration test if we want to rely on the activation to show the sequence partially activated. Using a unit test would block for 60 seconds while the first action activated as the action doesn't actually get invoked in a unit test, but instead times out.

Reply:
I'd think something like this will work https://github.com/apache/incubator-openwhisk/blob/acbef6aa714f65b29c5fe67b68c7fd2ffbce159a/tests/src/test/scala/whisk/core/controller/test/ActionsApiTests.scala#L1239 (create a mock active ack). I didn't try so may be missing a detail why it doesn't.

Reply:
Fancy, fancy. :)